guides/advanced_guides/tokenization.md

```diff
@@ -17,28 +17,39 @@ When you add documents to a MeiliSearch index, the tokenization process is handl
 We can break down the tokenization process like so:
 
 1. Crawl the document(s) and determine the primary language for each field.
-2. Go back over the documents field-by-field, running the corresponding tokenization pipeline, if it exists. Pipelines include many language-specific operations.
+2. Go back over the documents field-by-field, running the corresponding tokenization pipeline, if it exists.
+
+Pipelines include many language-specific operations. Currently, we have two pipelines:
+
+1. A specialized Chinese pipeline using [Jieba](https://github.com/messense/jieba-rs)
+2. A default pipeline that uses [unicode text segmentation](https://github.com/unicode-rs/unicode-segmentation).
```

Comment:
We don't use unicode segmenter yet, for now this is the same tokenizer as before, that basically separate words based on categories (separators and words components) to put things shortly.